uwblueprint/supportive-housing
# Get Tags Route

## Notion task link


[Get Tags Route](https://www.notion.so/uwblueprintexecs/2792c3504d744d2b90ff967c32d00961?v=d33629815ecf477ab4e1ff4e33774c32&p=cddd8fc25bd441c598811798a7847550&pm=s)



## Implementation description
* created a new get tags route that returns a list of all tags from the db
* created a new get tags service and interface



## Steps to test
1. You can try running the backend locally and try calling the URL: http://localhost:8080/tags through postman (as an unauthed user, it should display: "error": "You are not authorized to make this request." and as an admin, it should display a list of tag objects)



## What should reviewers focus on?
* Focus on the architecture and the flow from controller to service layer


## Checklist
- [] IF I have made changes to the db/models, I have updated the [Data Models Page](https://www.notion.so/uwblueprintexecs/Data-Models-760f8aa06b244eb0842c079ad77987b0)
